src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/book-store
+spring.datasource.username=root
+spring.datasource.password=Root1234!
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQLDialect
 
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
+
+spring.jackson.deserialization.fail-on-unknown-properties=true
```

Comment:
Why do you need this property?

Reply:
when the req body comes with unknown params, then Jackson will return a 400 code response

Reply:
which status will be returned without this property? I guess the same

Reply:
it'll just proceed if other data is valid. I don't know much about it, it was just recommended on the mate course